service-workers/cache-storage/resources/cache-keys-attributes-for-service-worker.js

```diff
@@ -0,0 +1,23 @@
+self.addEventListener('fetch', function(event) {
```

Comment:
nit: why not arrow function (`=>`) here?

Reply:
Done.